### PR TITLE
Tabnet visualization

### PR DESCRIPTION
**Pull request by longtle**
We are going to release TabNet as a built in algorithm on GCP platform: 
We provide a simple visualization so customer can understanding the output.

### CODE REVIEW

**Comment by sirtorry** [comment]
please change to 
PROJECT_ID = "[your-project-id]"
BUCKET_NAME = "[your-bucket-name]"

File: notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
+{
```
Comment (sirtorry) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>please don't declare bucket name twice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Comment by longtle** [comment]
Thanks, Torry.
I fixed as your comments and the format issues.

**Comment by sirtorry** [comment]
@longtle thank you so much for dealing with all of the checks

File: notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
```
Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Can you move these sentences down into the "Overview" section below? </p><p> </p><p>Also, rewrite as:</p><p> </p><p>AI Platform provides a built-in algorithm based on [TabNet](https://arxiv.org/abs/1908.07442).</p><p>Learn how to [train and deploy a model with the TabNet built-in algorithm](https://cloud.google.com/ai-platform/training/docs/algorithms/tab-net-start).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Can you add a link for "Synthetic 2 data output" and also a brief explanation of what that is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Change section title to "Reading a sample TabNet prediction" or something similar.</p><p> </p><p>"After training and serving your model, you upload the output to Cloud Storage. There is a sample TabNet prediction using synthetic data: </p><p>"</p><p> </p><p>Also explain a bit about aggregated mask values here - what are they, and what are we using them for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Where did you call <code>json.loads()</code>? Can you show how to parse the prediction using <code>json.loads()</code>  to show just the predicted classes, after you show the entire prediction?</p><p> </p><p>Change section title to "Download and parse the sample prediction" or something similar. That will make it clearer what the blob is, and what it's for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Can you add context here - explain that you're putting together all the mask values (from the entire provided sample prediction?) into a matrix to prepare it for visualization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Can you remove "The lighter color indicates more important feature." from the section title?</p><p>Instead, please add a sentence or two explaining how to interpret this plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (longtle) [reply notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Yes, I call it in the next section. I move parse it using json.loads() method to the next header.</p><p>The </p>

**Comment by longtle** [comment]
Thank you for your suggestions.

File: notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
```
Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>"TabNet, which is helpful in explaining the algorithm"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>"Download and preprocess the predictions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb:1]: <p>Parse and concatenate mask values</p><p> </p><p>"feature important" --&gt; "feature importance"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Pull request merged**